dbms/src/DataStreams/LimitBlockInputStream.h

```diff
@@ -46,7 +47,10 @@ class LimitBlockInputStream : public IProfilingBlockInputStream
 
 private:
     LoggerPtr log;
-    LocalLimitTransformAction action;
```

Comment:
This actually remove our current implementaion of `LimitBlockInputStream`, I think it is not recommended, cc @SeaRise

Reply:
`LocalLimitTransformAction` now is only used in `LimitBlockInputStream`, So class `LocalLimitTransformAction` also needs to be deleted.

Reply:
Removed.